pkg/compactor/compactor.go

```diff
@@ -127,7 +127,7 @@ type BlocksCompactorFactory func(
 ) (compact.Compactor, compact.Planner, error)
 
 // CompactorLimits defines limits used by the Compactor.
-type CompactorLimits interface {
+type CompactorLimits interface { //nolint
```

Comment:
Why?  If it was complaining about stuttering, consider naming it `compactor.Limits`.

Reply:
Agree, we shouldn't have `//nolint` I will update the name to `Limits`